senders/interfaces.go

```diff
@@ -3,53 +3,10 @@
 package senders
 
 import (
-	"github.com/wavefronthq/wavefront-sdk-go/event"
-	"github.com/wavefronthq/wavefront-sdk-go/histogram"
-	"github.com/wavefronthq/wavefront-sdk-go/internal"
+	"github.com/wavefronthq/wavefront-sdk-go/wavefront"
 )
 
-// MetricSender Interface for sending metrics to Wavefront
-type MetricSender interface {
```

Comment:
Again, breaks backwards compatibility. Let's deprecate and use aliases if needed.